index.bs

```diff
@@ -2789,6 +2789,50 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats ### {#signature-attestation-types}
+    - For COSEAlgorithmIdentifier -7 (ES256), `sig` contains a pair of 32-byte integers,
```

Comment:
The `r` and `s` values might not be 32 bytes long in this encoding, and I think the reference for the encoding is [RFC 3278 section 3.3.2](https://tools.ietf.org/html/rfc3279#section-2.2.3). So I've say something like:

For {{COSEAlgorithmIdentifier}} -7 (ES256), and other ECDSA-based algorithms, a signature is encoded as an ASN.1 DER `Ecdsa-Sig-Value`, as defined in [[RFC3279]] section 3.3.1. 

(Then drop the added reference to X9.62.)

Reply:
Thanks